src/Detector/Detector.csproj

```diff
@@ -57,7 +57,9 @@
     <PackageReference Include="Microsoft.CSharp" Version="4.7.0" />
     <PackageReference Include="Microsoft.Extensions.Http" Version="7.0.0" />
     <PackageReference Include="Newtonsoft.Json" Version="13.0.2" />
+    <PackageReference Include="runtime.unix.System.Private.Uri" Version="4.3.2" />
```

Comment:
nit: I'm not sure if we need to include these package references in `Detector.csproj` and `Detector.Test.csproj` since they already reference the `Common.csproj` project, so these package references should be propagated downstream to these projects, but it's worth double-checking this

Reply:
Please have a look on the comment: https://github.com/microsoft/Oryx/pull/2068#issuecomment-1640737635